coalib/output/ConsoleInteraction.py

```diff
@@ -9,7 +9,7 @@
 try:
     # This import has side effects and is needed to make input() behave nicely
     import readline  # pylint: disable=unused-import
-except ImportError:  # pragma: no cover
+except ImportError:  # pragma posix: no cover
```

Comment:
This is causing problems on Windows, as we use coala-utils, and it includes pyreadline on Windows, so this never fails.

https://gitlab.com/coala/coala-utils/commit/43c81871476e14ce0d777129bff8cdfc557be342

possibly we need tests like were written for https://github.com/coala/coala/pull/5645